src/Core/Grid/Query/LogQueryBuilder.php

```diff
@@ -222,9 +222,9 @@ private function applyFilters(array $filters, QueryBuilder $qb)
     /**
      * @return string
      */
-    private function getEmployeeField(bool $includeFirstname = true): string
+    private function getEmployeeField(bool $includeFullFirstname = true): string
     {
-        if ($includeFirstname) {
+        if ($includeFullFirstname) {
```

Comment:
@atomiix why full first name? first name plus last name = full first name? ;)

Reply:
@kpodemski  beause if `false` it will return the` first letter of the firstname + lastname`

```php
return 'CONCAT(LEFT(e.`firstname`, 1), \'. \', e.`lastname`)';
```

Reply:
👍🏻

haven't noticed on mobile